test/RachisTests/DatabaseCluster/ReplicationTests.cs

```diff
@@ -1460,5 +1462,97 @@ public async Task ReplicateRaftDocuments()
                     TimeSpan.FromSeconds(10)));
             }
         }
+
+        [Fact]
+        public async Task ServerWideExternalReplicationShouldWork_NonShardedToSharded()
```

Comment:
let's move those tests to ShardedExternalReplicationTests or something

Reply:
Done